code/game/gamemodes/changeling/changeling_powers.dm

```diff
@@ -962,3 +962,71 @@ var/list/datum/dna/hivemind_bank = list()
 
 	feedback_add_details("changeling_powers","ED")
 	return 1
+	
+mob/proc/changeling_extract_dna_sting()
+	set category = "Changeling"
+	set name = "Arm Blade (20)"
+	set desc="Transform your arm into a deadly blade. Use it again to retract the blade."
+	
+	genetic_damage = 10
+	req_human = 1
+	max_genetic_damage = 20
+	weapon_type = /obj/item/weapon/melee/arm_blade
+	weapon_name_simple = "blade"
+
+/obj/item/weapon/melee/arm_blade
+	name = "arm blade"
+	desc = "A grotesque blade made out of bone and flesh that cleaves through people as a hot knife through butter"
+	icon = 'icons/obj/weapons.dmi'
+	icon_state = "arm_blade"
+	item_state = "arm_blade"
+	flags = ABSTRACT | NODROP
+	w_class = 5.0
+	force = 25
+	throwforce = 0 //Just to be on the safe side
+	throw_range = 0
+	throw_speed = 0
+	sharpness = IS_SHARP
+
+/obj/item/weapon/melee/arm_blade/New(location,silent)
+	..()
```

Comment:
Should there not be a check for whether or not the mob is a monkey somewhere around here?